.snyk

```diff
@@ -0,0 +1,8 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.14.1
+ignore: {}
+# patches apply the minimum changes required to fix a vulnerability
+patch:
+  SNYK-JS-LODASH-567746:
+    - lodash:
+        patched: '2020-05-11T20:10:49.155Z'
```

Comment:
I don't really love this sort of monkey patching. How sever is the vulnerability and is there an issue on lodah that has traction to fix it?

Reply:
Yeah, not a big fan either.  Here's what I know:

- Medium severity but *can* be attacked over the network (JSON deserialization potentially) : "The function `zipObjectDeep` can be tricked into adding/modifying properties of the Object prototype

- zipObjectDeep is only referenced in the documentation of jsforce.js and jsforce-core.js but is not directly used in any jsforce calls.

- PR: https://github.com/lodash/lodash/pull/4759 (much begging for a release) and alternative PR https://github.com/lodash/lodash/pull/4745

I'm at a bit of a loss regarding the right way forward given the lack of a patched lodash release.

However, I haven't been able to put together an experiment to determine what happens in /downstream/ projects that now pull in unpatched lodash as a transitive dependency - I think they'll need to have the same ignore/suppress setting in their .snyk file.

Reply:
lodash finally merged the above PR and released a new version so this specific monkey patch for lodash can be removed.

Reply:
Removed .snyk bits since `lodash` updated and monkey patching no longer needed.